### PR TITLE
app-office/calcurse: add 4.7.1 and fix documentation

### PR DESCRIPTION
**Pull request by kobboi**
calcurse 4.7.1 is a bug-fixing release.

Also updated PYTHON_COMPAT (used by /usr/bin/calcurse-{vdir,caldav}).

Thanks-to: Marco Sirabella <marco@sirabella.org>
Closes: https://bugs.gentoo.org/813438
Signed-off-by: Christophe Lermytte <gentoo@lermytte.be>

### CODE REVIEW

File: app-office/calcurse/calcurse-4.7.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	"${FILESDIR}"/${PN}-4.2.1-tinfo.patch
+	"${FILESDIR}"/${PN}-4.7.1-respect-docdir.patch
+)
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+
```
Comment (thesamesam) [inline app-office/calcurse/calcurse-4.7.1.ebuild:39]:
```suggestion
```

File: app-office/calcurse/calcurse-4.7.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+inherit autotools python-single-r1
+
+DESCRIPTION="A text-based calendar and scheduling application"
+HOMEPAGE="https://calcurse.org/"
+SRC_URI="https://calcurse.org/files/${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="amd64 ppc ppc64 x86"
```
Comment (thesamesam) [inline app-office/calcurse/calcurse-4.7.1.ebuild:15]: ~arch (```ekeyword \~all foo.ebuild```)

File: app-office/calcurse/calcurse-4.7.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{8..10} )
```
Comment (thesamesam) [inline app-office/calcurse/calcurse-4.7.1.ebuild:5]:
```suggestion

PYTHON_COMPAT=( python3_{8..10} )
```

Nit: new line after EAPI

Comment (kobboi) [reply app-office/calcurse/calcurse-4.7.1.ebuild:6]: Resolved

**Pull request closed without merging**